### ISSUE TITLE
Version 1.8.0 causing status undefined

### ISSUE THREAD

**Issue by herjunop-sprout** [adaltas/node-http-status#51]
**Describe the bug**

Package version 1.8.0 causing status undefined

**Additional context**

<img width="714" alt="image" src="https://github.com/user-attachments/assets/c12ea1d9-72c1-41bc-b722-bb5cc9b87d41">

**Comment by wdavidw** [comment]
Could you share the code generating this error. We changed the build and it will help to know how the error is generated.

**Comment by bensquire** [comment]
Also seeing issues after having upgrading to 1.8.0.

Could this be related to the change to ESM? For example, in our TS project for this import:

```
import { BAD_REQUEST, FORBIDDEN } from 'http-status'
```

We now get the error:

```
TS2307: Cannot find module http-status or its corresponding type declarations.
There are types at
TRUNCATED/node_modules/http-status/dist/index.d.ts, but this result could not be resolved under your current moduleResolution setting. Consider updating to node16, nodenext, or bundler
```

**Comment by aferrando-tid** [comment]
Yes, looks like 1.8.0 should have been a major, not a minor

**Comment by realtebo** [comment]
Has someone a workaround ?
Actually we downgraded to 1.7
But I think, if package has been published, it is usable, so ... how?

**Comment by aferrando-tid** [comment]
1.8.1 that would revert to the previous (in case something else aside from the migration to ESM was added), and then a 2.0.0 with the ESM change

**Comment by jchuder** [comment]
Update my project's version from "^1.6.2" to "1.6.2" in package.json and remove node_module then do yarn install to workaround fixed this issue.

> Looks like root cause is 1.8.0 change the structure of package.
> ```javascript
> const HTTPStatus = require('http-status');
> 
> // Old structure
> console.log(HTTPStatus.OK); // 200
>
> // New structure
> console.log(HTTPStatus.OK); // undefined
> console.log(HTTPStatus.default.OK); // 200
> ```

**Comment by omob** [comment]
As suggested by @jchuder, removing the caret works. This will prevent fetching the latest 1.8.0 build that has the incompatibility issue.

**Comment by wdavidw** [comment]
Two new versions are published:
- version 1.8.1 fixes 1.8.0 and is a copy of the previous version 1.7.4
- version 2.0.0 is the new version with the convertions to ESM and TypeScript

**Comment by wdavidw** [comment]
Closing, please open a new issue if you encounter problems with version 2.
